src/Docs/new-command.md

```diff
@@ -58,18 +58,16 @@ This keeps all code, options, models, and tests for a service together. See `src
    - `operation`: Action to perform (verb, lowercase)
 
    Each command is:
-   - In code, to avoid ambiguity between service classes and Azure services, we
-     refer to Azure services as Areas
+   - In code, to avoid ambiguity between service classes and Azure services, we refer to Azure services as Areas
    - Registered in the RegisterCommands method of its service's Areas/{Area}/{Area}Setup.cs file
    - Organized in a hierarchy of command groups
    - Documented with a title, description and examples
    - Validated before execution
    - Returns a standardized response format
 
-   **IMPORTANT**: Command group names cannot contain dashes. Use camelCase or concatenated names instead:
+   **CRITICAL**: Command group names can contain dashes. Use camelCase or concatenated names instead:
```

Comment:
'Command group names can contain dashes.'

isn't critical. :)

We can just remove the section.

Reply:
If I understand correctly, w/ the change, now underscore cannot be used in command group names, right?

Reply:
The ciritial I put is due to breaking change. I could remove it or rename it for this section. 

via reading the contribuation markdown file, it seems this new-command.md is not only for contributor, but also for the model to generate the code. Do we still need to clarify here to let model know the rules?

Reply:
Yes. After the change, we need to clarify that underscores are prohibited in command group names. (Please correct me if I am wrong)